src/Orchard.Web/Modules/Orchard.Modules/Controllers/AdminController.cs

```diff
@@ -155,17 +172,19 @@ public class AdminController : Controller {
                 return HttpNotFound();
             }
 
-            try {
-                _recipeManager.Execute(recipe);
+            var executionId = _recipeManager.Execute(recipe);
+
+            if (String.IsNullOrEmpty(executionId))
+            {
+                Logger.Error("Error while executing recipe {0} in {1}.", moduleId, name);
+                Services.Notifier.Error(T("Error while executing recipe {0} in {1}.", moduleId, name));
             }
```

Comment:
I think the parameters are reversed on these two (moduleId vs. name)? I see this was the case also in the old code, but now it's also for the notifier, so we might as well fix it.